bedrock/firefox/templates/firefox/whatsnew-base-resp.html

```diff
@@ -0,0 +1,18 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-resp.html" %}
+
+{#- The firstrun and whatsnew pages should not be indexed by search engines -#}
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {% if l10n_has_tag('meta_update_201503') %}
+    {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
```

Comment:
To be completely honest, I don't like the approach. We can add the string to main.lang, and then we're stuck next time we decide to update this string because we want to focus communication on something else.

I'd prefer what's new page/first run to be threated individually, and add string as necessary. I'd also like to see numbers on how many people actually share the whatsnew page (do we have them?), because it really seems an edge case.